cypress/e2e/po/pages/extensions-compatibility-tests/extensions-compatibility.po.ts

```diff
@@ -0,0 +1,98 @@
+import PagePo from '@/cypress/e2e/po/pages/page.po';
+import { InstallChartPage } from '@/cypress/e2e/po/pages/explorer/charts/install-charts.po';
+import Kubectl from '@/cypress/e2e/po/components/kubectl.po';
+import ChartInstalledAppsPagePo from '@/cypress/e2e/po/pages/chart-installed-apps.po';
+import ProductNavPo from '@/cypress/e2e/po/side-bars/product-side-nav.po';
+import BaseResourceList from '@/cypress/e2e/po/lists/base-resource-list.po';
+import CodeMirrorPo from '@/cypress/e2e/po/components/code-mirror.po';
+import AsyncButtonPo from '@/cypress/e2e/po/components/async-button.po';
+import LabeledInputPo from '@/cypress/e2e/po/components/labeled-input.po';
+import CheckboxInputPo from '@/cypress/e2e/po/components/checkbox-input.po';
+
+const installChart = new InstallChartPage();
+const terminal = new Kubectl();
+const installedApps = new ChartInstalledAppsPagePo();
```

Comment:
just in case this makes it after refactor, it's missing required args (ts error)